internal/pkg/secret/encryption_other.go

```diff
@@ -25,11 +25,10 @@ const (
 
 func GenerateRandomBytes(n int) ([]byte, error) {
 	b := make([]byte, n)
-	_, err := rand.Read(b)
-	if err != nil {
+	if _, err := rand.Read(b); err != nil {
 		return []byte{}, err
 	}
-	return b, err
+	return b, nil
```

Comment:
Thanks! Lines like this one could've caused an unintended bug.

Reply:
Yup. I just happened to notice this while changing other stuff. I wonder if there's some regex string we could write to find these.